tests/functional/s3cmd/configure.bash

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 apt-get install -y -q  python-dateutil python-magic
-wget http://launchpadlibrarian.net/222422124/s3cmd_1.6.0-2_all.deb
-dpkg -i s3cmd*.deb
+wget https://launchpad.net/ubuntu/+archive/primary/+files/s3cmd_1.6.1.orig.tar.gz
```

Comment:
~~This can be updated to use the `pip install` command.~~
I think this file is legacy and can be safely removed. We have to do the same for other test dirs. as well (may/may not be in this PR).